instrumentation-test/src/main/java/com/newrelic/agent/introspec/internal/ExternalRequestImpl.java

```diff
@@ -23,15 +25,19 @@
 
     private String library;
     private String operation;
+    private Integer statusCode;
```

Comment:
So do these need to be mutable after construction?  If not, can we make them final?